Ogury/build.gradle

```diff
@@ -0,0 +1,8 @@
+project.version = '5.0.7.1'
```

Comment:
**suggestion** (non-blocking): The initial version of our adapters should be 0, not 1.
```suggestion
project.version = '5.0.7.0'
```

Reply:
Fixed in ecdb97582edd887a4ba69ae77ba6a6bd05a8d402.